src/module/actor/modifiers.ts

```diff
@@ -140,6 +140,13 @@ class ModifierPF2e implements RawModifier {
     notes: string;
     hideIfDisabled: boolean;
 
+    unidentified: boolean;
+    originalLabel: string;
+
+    static get UNIDENTIFIED() {
+        return game.i18n.localize("PF2E.identification.Unidentified");
+    }
```

Comment:
Is there a reason for this to be a getter?

Reply:
If i remember correctly, my thought process was that at class definition, `game.i18n` may not exist, but i didn't do any test on that, the fact that it is a getter itself makes it easy to change into a property at any time.

Reply:
You can just localize this as needed. No need for a micro-optimization here.

Reply:
But everywhere it is used it is in need of `localization` event in the class constructor itself.